app/views/patients/update.js.erb

```diff
@@ -0,0 +1,25 @@
+// $('a[href$="#patient_information"]').click();
+
+$('#patient_dashboard').html(
+  '<%= escape_javascript(render partial: "patients/patient_dashboard", locals: { patient: @patient })%>'
+);
+
+// $('#change_log').remove();
+$('#change_log').html(
+  '<%= escape_javascript(render partial: "patients/change_log", locals: { patient: @patient })%>'
+);
+
+// $('#fulfillment').remove();
+<% if @patient.previous_changes.key?('pledge_sent') && current_user.allowed_data_access? %>
```

Comment:
the idea here is that if pledge sent changed, we rerack the particular partials dealing with it if a user can see them. That includes nuking or populating the fulfillment form if necessary.